libraries/stdlib/samples/test/samples/random/randoms.kt

```diff
@@ -38,4 +38,65 @@ class Randoms {
         // random with another seed produce another sequence
         assertPrints(randomValues3, "[14, 48, 57, 67, 82, 7, 61, 27, 14, 59]")
     }
-}
\ No newline at end of file
+
+    @Sample
+    fun nextBits() {
+        // always generates a 0
+        println(Random.nextBits(0))
+        // get a random 1 bit value
+        println(Random.nextBits(1))
+        // get a random 8 bit value
+        println(Random.nextBits(8))
+        // get a random 16 bit value
+        println(Random.nextBits(16))
+        // get a random 32 bit value
+        println(Random.nextBits(32))
+    }
+
+    @Sample
+    fun nextBoolean() {
+        // get a random Boolean value
+        println(Random.nextBoolean())
```

Comment:
Samples are meant to clarify function behavior and provide usage examples for documentation readers. This sample do not seem to comply the criteria.

Consider writing more or less realistic use cases. For example:
```
val isAliceFirst = Random.nextBoolean()
if (isAliceFirst) {
    println("Alice goes first.")
} else {
    println("Bob goes first.")
}
```